packages/datadog-instrumentations/src/helpers/hooks.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
   'body-parser': () => require('../body-parser'),
   'bunyan': () => require('../bunyan'),
   'cassandra-driver': () => require('../cassandra-driver'),
+  'child_process': () => require('../child-process'),
```

Comment:
ok, added.
Not 100% necessary yet, because it looks like the tracer is already requiring `child_process` in different points. But it could change in the future, so I've added it.
Thanks